docs/application/dotnet/guides/multimedia/roi-tracker.md

```diff
@@ -0,0 +1,67 @@
+# Roi Tracker
+
+[RoiTracker](/application/dotnet/api/TizenFX/latest/api/Tizen.Multimedia.Vision.RoiTracker.html) is a new feature of Media Vision API since Tizen 7.0(C# API10). This API allows users to obtain the proper ROI coordinates that users want to track in an image. For example, when an image and ROI coordinates are provided as input to this API, the Media Vision framework will process the given image and ROI coordinates from the decoded image data and will provide ROI coordinates within the given image.
```

Comment:
```suggestion
ROI tracker is a new feature of the Media Vision API since Tizen 7.0 (C# API10). The [RoiTracker](/application/dotnet/api/TizenFX/latest/api/Tizen.Multimedia.Vision.RoiTracker.html) API allows users to obtain the proper ROI coordinates that users want to track in an image. For example, when an image and ROI coordinates are provided as input to this API, the Media Vision framework will process the given image and ROI coordinates from the decoded image data and will provide ROI coordinates within the given image.
```